src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/TransactionServiceImpl.java

```diff
@@ -156,6 +160,9 @@ public void updatePendingTransactions(final List<TransactionUpdateRequestDto> tr
 	public void saveDeposits(final LotDepositRequestDto lotDepositRequestDto, final TransactionStatus transactionStatus) {
```

Comment:
Can we make a unit test for saveDeposits, considering the test case that source study may or may not have been specified?

Reply:
Done